addons/web_editor/static/src/js/editor/odoo-editor/src/style.css

```diff
@@ -17,6 +17,59 @@
     background-color: rgba(117, 167, 249, 0.5) !important; /* #bad3fc equivalent when over white*/
     cursor: pointer !important;
 }
+.oe-floating {
+    box-shadow: 0px 3px 18px rgba(0, 0, 0, .23);
+    border-radius: 4px;
+    position: absolute;
+}
+
+/* table ui styling */
+
+.o_table_ui {
+    background-color: transparent;
+    position: absolute;
+    z-index: 10;
+    padding: 0;
+}
+.o_table_ui.o_open {
+    visibility: visible !important;
+}
+.o_table_ui.o_open .o_table_ui_menu {
+    display: block;
+}
+.o_table_ui > div {
+    position: absolute;
+}
+.o_table_ui_menu_toggler {
+    cursor: pointer;
+    background-color: white;
+    color: #6e727a;
+    border: 1px solid #C9CCD2;
+    border-radius: 5px;
+    padding: 2px 3px;
+}
+.o_table_ui_menu {
+    display: none;
+    cursor: pointer;
+    background-color: white;
+    width: fit-content;
+    border: 1px solid #C9CCD2;
+    padding: 5px 0;
+    white-space: nowrap;
+}
+.o_table_ui_menu > div:hover {
+    background-color: #ebebeb;
+}
+.o_table_ui.o_open .o_table_ui_menu > div.o_hide {
+    display: none;
+}
+.o_table_ui_menu span {
+    margin-right: 8px;
+    color: #6e727a;
+}
+.o_table_ui_menu div {
+    padding: 0 8px;
+}
```

Comment:
Most rules apply to `o_table_ui` or `otable_ui_menu`. I suggest to consolidate these rules into groups.

Reply:
We have SCSS now also so this will be nicer.